### DISCUSSION TITLE
Small timer Quality of Life improvement

### DISCUSSION THREAD

**Discussion by TaddyJ** [babybuddy/babybuddy#853] [Ideas]
Could I request that a timer that has had its title edited carry that title into the timer editing (final) page?
It could be

Example.
My wife and I both feed our kids.
Sometimes one of us weighs the milk, starts a timer and titles it with the amount it weighs. Hands off the feed and the other finishes the feed.
Its easy to identify the timer set because it was named (Sometimes we have multiple timers for our other kids sleeping, expressing, feeding etc).
Once we open it and then start to edit it to finish it. The name given in the initial timer setup has gone. So we have to either go back and double check or try and remember. 
Sometimes night feeds when sleepy are hard to remember.

**TLDR: If an ongoing timer is edited to have a name. Please add that edited name to the finish timer screen somewhere.**

Example
![image](https://github.com/user-attachments/assets/4ace3d73-697b-4b8e-8594-889049a4186a)

**Comment by cdubz** [comment]
Hmmmm yes that’s doable. Will just to figure where to add it.

**Reply by TaddyJ** [reply]
Thankyou! 
I know its not a feature set or bug. But I think it would be a very nice improvement to an occasional use case with no drawbacks. :)

**Reply by TaddyJ** [reply]
Hey. Haven't seen much action on this. 
While using it and editing existing timers I see the "Timer Name" is already displayed. Its only when you select one of the 4 options the name disappears.

For example here is 2x timers I have running as a demo..
![image](https://github.com/user-attachments/assets/d0135f2c-7a27-4d7c-89b5-89126a927530)
Can click on either to bring it up and you can see the name carries over in multiple locations.
![image](https://github.com/user-attachments/assets/2b88dd8a-635a-4ba1-88a6-70746ab418d9)
If I hit the edit button it still carries over..
![image](https://github.com/user-attachments/assets/03660fde-30fd-4709-88cb-6fbd88fd907e)
But now when I select one of the 4 options from the previous screen (for example feeding because we put the amount of EBM we are feeding our daughter in the timer name) the timer name does not carry over.. 
![image](https://github.com/user-attachments/assets/5e41bb6e-07ef-48d5-b897-609b8d734cd0)
I'm not a programmer or web designer but given the other screens I figured it would be easy to implement it? 
Would be helpful to remember the title because we often have several running at once for our 2 kids for sleeps/feeds etc..

I appreciate all the work and Hopefully see something on this soon.

**Reply by cdubz** [reply]
@TaddyJ I created issues #914 for better visibility on this.
